src/hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+
+export const useLocalStorage = <T extends string>(
```

Comment:
I'm constraining the type of useLocalStorage to a string for now, but it would be a good idea to let this be more flexible since we can technically store anything in local storage.

Reply:
Makes sense, in case in the future for whatever reason we wanted a boolean, etc.